packages/react-app/src/LineChart.js

```diff
@@ -0,0 +1,201 @@
+import React, {Component} from "react";
+import "./LineChart.css";
+
+
+class LineChart extends Component {
```

Comment:
Nice! This is cool, wondered why you didn't use a charting library?

Reply:
I already worked with this one so decided to choose it here. Is that okay? 😅